src/compression_with_clause.c

```diff
@@ -50,8 +50,9 @@ throw_segment_by_error(char *segment_by)
 {
 	ereport(ERROR,
 			(errcode(ERRCODE_SYNTAX_ERROR),
-			 errmsg("unable to parse the compress_segmentby option '%s'", segment_by),
-			 errhint("The compress_segmentby option should be a comma separated list of column "
+			 errmsg("unable to parse the timescaledb.compress_segmentby option '%s'", segment_by),
```

Comment:
hmm i think the "the" should be dropped here

Reply:
fixed